R/zzz.R

```diff
@@ -32,3 +32,7 @@ setdiff_teal_slices <- getFromNamespace("setdiff_teal_slices", "teal.slice")
 coalesce_r <- getFromNamespace("coalesce_r", "teal.slice")
 # all *Block objects are private in teal.reporter
 RcodeBlock <- getFromNamespace("RcodeBlock", "teal.reporter") # nolint
+
+# Use non-exported function(s) from teal.code
+# This one is here because format_expression should not be exported from teal.code
+format_expression <- getFromNamespace("format_expression", "teal.code")
```

Comment:
This will be repaced with `lang2calls` when we merge the `refactor` branch due to https://github.com/insightsengineering/teal.code/pull/176

Reply:
No reason to not use it right now, done in a861bec97